lib/explorer/inspect.ex

```diff
@@ -59,6 +59,33 @@ defmodule Explorer.Inspect do
         2
       )
 
+  def df_pretty_inspect(df, nrow \\ 5) do
+
+    df = Explorer.DataFrame.slice(df, 0, nrow)
+
+    types = df
+      |> Explorer.DataFrame.dtypes()
+      |> Enum.map(&Atom.to_string(&1))
+      |> Enum.map(fn x -> "<" <> x <> ">" end)
+
+    headers = Explorer.DataFrame.names(df)
+
+    values = headers
+      |> Enum.map(&Explorer.DataFrame.pull(df, &1))
+      |> Enum.map(&Explorer.Series.to_list(&1))
+      |> Enum.zip()
+      |> Enum.map(&Tuple.to_list(&1))
+
+    TableRex.Table.new()
+      |> TableRex.Table.put_header(headers)
+      |> TableRex.Table.add_row(types)
+      |> TableRex.Table.add_rows(values)
+      |> TableRex.Table.render!(
+          header_separator_symbol: "=",
+          horizontal_style: :all)
+      |> IO.puts
+  end
```

Comment:
Btw, it is probably too early, but don't forget to run `mix format` to get formatted code later. :)